src/test/java/ReporterTest.java

```diff
@@ -0,0 +1,56 @@
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/*
+ * This test class is meant to test that the reporter outputs the appropriate
+ * information in relation to the ouptut from SB2 and the given test JSON
+ * file.
+ * 
+ * The current output I'm expecting (VERY subject to change)  will be
+ * Sprites: #
+ *  - SpriteName
+ *     - Length: #
+ *  - SpriteName
+ *     - Length: #
+ * This is important to note to be able to better understand the tests below.
+ *
+ * NOTE: If we could extract a student's name from the file name it would be
+ * really handy to have the student's name at the top of the file for teacher's
+ * review purposes.    
+ */
```

Comment:
I think that this is a fine format, but it could be annoying to manage the markdown-like indentation. Do you think it's worth bringing in a file generator for YAML or Markdown?